SPECS/audit/audit.spec

```diff
@@ -152,6 +153,9 @@ echo "disable auditd.service" > %{buildroot}%{_libdir}/systemd/system-preset/50-
 %{python3_sitelib}/*
 
 %changelog
+* Fri Dec 10 2021 Chris Co <chrco@microsoft.com> - 3.0.6-1
```

Comment:
Adding a note for why we need to update audit to 3.0.6:
Current audit 3.0 version fails to build because linux/ipx.h header
is no longer part of 5.15 kernel source. audit 3.0.6 has a change to
handle this 5.15 difference.